test/unit/math/prim/scal/meta/is_fvar_test.cpp

```diff
@@ -0,0 +1,8 @@
+#include <stan/math/prim/scal.hpp>
+#include <gtest/gtest.h>
+
+TEST(MetaTraits, is_fvar) {
+  using stan::is_fvar;
+  EXPECT_FALSE(is_fvar<int>::value);
+  EXPECT_FALSE(is_fvar<double>::value);
```

Comment:
Would you mind making sure this doesn't return true for `var` with a test?

Reply:
The test should be in the rev directory. The reason is that the includes are different and there might be a specialization included in rev that’s not included in prim or fwd.

Reply:
The test should be in the rev directory. The reason is that the includes are different and there might be a specialization included in rev that’s not included in prim or fwd.

Reply:
Agreed that if there is a test for `is_fvar<var>` that that should go in `rev`. Didn't mean to say we should add it to this file.

Reply:
Added: https://github.com/stan-dev/math/blob/2e9b0377725f1b854332d04d6e1da8ddd3927831/test/unit/math/rev/scal/meta/is_fvar_test.cpp